.taskcluster.yml

```diff
@@ -36,7 +36,7 @@ tasks:
           && git config advice.detachedHead false
           && git checkout {{event.head.sha}}
           && echo "--" > .adjust_token
-          && ./gradlew clean assemble lint checkstyle ktlint pmd test
+          && ./gradlew clean assemble lint checkstyle ktlint pmd test detektCheck
```

Comment:
nit: please run this before `test`: afaik, test uses the most CPU time so it'd be great if we can stop builds before it runs. Here and below.